### ISSUE TITLE
mac osx - application hangs open indefinitely after jar has exited

### ISSUE THREAD

**Issue by howaldmg** [libgdx/packr#145]
I'm working with a very standard build (nothing fancy or exotic).  The exe produced for windows doesn't seem to exhibit this behavior.

When running the application as a bundle or a standalone binary, the jar starts up, does what it's supposed to do, and then exits beautifully.  The application that started the jar hangs open, however.

I noticed this problem when starting the application from the terminal, but it also happens when the application is double-clicked in finder.  Once it starts to hang, my only option is to force quit it or (kill -9).  I'm running on macOS Mojave, version 10.14.5.

Any workarounds would be welcome.

**Comment by howaldmg** [comment]
I did some digging and found a code example that looks to have served the basis of the MacOS implementation.  https://developer.apple.com/library/archive/technotes/tn2147/_index.html

In the 'simplejavalauncher' example that can be downloaded, when DestroyJavaVM is called, `exit(result);` gets called before returning null.  I haven't tried the fix myself yet, (setting up a dev environment for packr isn't something I budgeted time for) but it might explain the application hanging.  

If anyone reading this could try it out and set up a PR if it works, I'd really appreciate it.  My work around is fine for now, but it's pretty hacky and could be problematic if my use case ever got any more complex.
